content/en/docs/components/katib/katib-config.md

```diff
@@ -0,0 +1,211 @@
++++
+title = "Katib Configuration Overview"
+description = "How to make changes in Katib configuration"
+weight = 40
+                    
++++
+
+This page describes information about
+[Katib config](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/katib-controller/katib-config.yaml).
+
+Katib config is the Kubernetes
+[Config Map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) that contains information about:
+
+1. Current [metrics collectors](/docs/components/katib/experiment/#metrics-collector) (`key = metrics-collector-sidecar`)
+1. Current [algorithms](/docs/components/katib/experiment/#search-algorithms-in-detail) (suggestions) (`key = suggestion`).
+
+Katib Config Map must be deployed in
+[`KATIB_CORE_NAMESPACE`](/docs/components/katib/env-variables/#katib-controller)
+namespace with `katib-config` name. Katib controller parses Katib config when you submit Experiment.
+
+You can edit this Config Map even after deploying Katib.
+
+If you deploy Katib in Kubeflow namespace, to edit Katib config run this:
+
+`kubectl edit configMap katib-config -n kubeflow`
+
```

Comment:
Suggestion - use code style/code font/code block/codefence (K8s style [guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-inline-code-commands-and-api-objects), Google style [guide](https://developers.google.com/style/code-in-text#formatting-a-command):

```suggestion

    ```shell
    kubectl edit configMap katib-config -n kubeflow
    ```
```
(without the tabs that indent the code block - they render differently if I remove them)

Reply:
@8bitmp3 Should we use ```shell everywhere?
e.g. https://github.com/kubeflow/website/blame/master/content/en/docs/components/hyperparameter-tuning/hyperparameter.md#L144 ?

Reply:
@8bitmp3 I've just seen you comment: https://github.com/kubeflow/website/pull/2320/files#r516300401 about it.
Will do the change in this PR.

Reply:
Done in https://github.com/kubeflow/website/pull/2312/commits/fe91e741a3f97bdd81e9e34b63a409124f9f933c.